charts/k8s-monitoring-v1/docs/examples/application-observability/metrics.alloy

```diff
@@ -116,12 +117,14 @@ otelcol.processor.transform "default" {
       "set(attributes[\"namespace\"], attributes[\"k8s.namespace.name\"])",
       "set(attributes[\"loki.resource.labels\"], \"cluster, namespace, job, pod\")",
```

Comment:
No longer relevant if we send OTLP logs to the Grafana Cloud OTLP Endpoint and adopt the new Loki format for OTLP logs.
`loki.resource.labels` will be ignored or, worse, added as a meaningless resource attribute ie a metadata.
This comment applies to all the files where this config block is added.